scripts/sendUserTxToErcForeign.js

```diff
@@ -67,6 +68,7 @@ async function main() {
       const gasLimit = await poa20.methods
```

Comment:
In order to be re-assigned, `gasLimit` should be declared with `let` instead of `const`